Anabi.DataAccess.Ef/AnabiContext.cs

```diff
@@ -83,5 +83,13 @@ public AnabiContext(DbContextOptions<AnabiContext> options) : base(options)
         public DbSet<CrimeTypeDb> CrimeTypes { get; set; }
 
         public DbSet<PrecautionaryMeasureDb> PrecautionaryMeasures { get; set; }
+        
+        protected override void OnModelCreating(ModelBuilder modelBuilder)
+        {
```

Comment:
Why is this necessary?

Reply:
I deleted that. It's just what .NET core shows.